### ISSUE TITLE
Simplified Intake Option for Internal Topcoder Projects

### ISSUE THREAD

**Issue by acshields** [appirio-tech/connect-app#4080]
### Update 1
When a Topcoder employee user clicks "+ New Project" the following should occur:
1. The top-level catalog screen should load (see image)
2. The Topcoder employee user should see a third option, which states "Internal Topcoder Project"
- This option should only be viewable to Topcoder employees
- The title should state: "Internal Topcoder Project"
- The description should state: "This option should only be used for internal Topcoder projects." 
3. When the Topcoder employee user selects this option, they should be routed directly into the intake form.

### Update 2
Create a simplified intake form, based upon these requirements:
- Screen 1 (Objective Flags)
Form Header: "Before we start"
Question: "Why are you creating this project today?"
Answer Format: [Radio Buttons, with response options below]
-Internal Project
-Demo/Test/Other
- Screen 2 (Primary Intake Form)
Form Header: Project Details
Question 1 (Required): Name your project
Question 1 Answer Format (Required): [Free Text]
Question 2 (Optional): Describe project goals.
Question 2 Answer Format (Optional): [Free Text]
At this stage, the intake form is now complete. The Topcoder employee user should be able to click a button that says "Create Project" and have the project generated.

Topcoder Employee Users who should see this option:
-Connect Manager
-Connect Admin
-Administrator
-Connect Account Manager
-Business Development Representative
-Presales
-Account Executive
-Program Manager
-Solution Architect
-Project Manager

Who should not see this intake option:
-Copilots
-Clients

**Comment by acshields** [comment]
@RishiRajSahu - I have added notes above about the users who should be able to view/use this simplified intake option.

**Comment by acshields** [comment]
@RishiRajSahu - in the section "Update 1" above, you can see the header copy & description that can display in the catalog.

**Comment by RishiRajSahu** [comment]
@acshields the changes are in dev for reviewing. Also, I have put TBD where the copy changes are needed.

FYI @lakshmiathreya

**Comment by acshields** [comment]
@RishiRajSahu - Just recapping our notes from the call.

- On the first catalog screen, update the displayed order so that "Solutions" and "Talent as a Service" are displayed next to each other at the top, with the "Internal Topcoder Project" option displayed below.
- For the Internal Topcoder Project, when the button is clicked EITHER:
-Just have the user routed directly into the form (bypassing the sub-catalog screen)
-Or, use the same copy from screen 1 of the catalog on screen 2 (sub-catalog) and remove the "Learn More" button, if this option is easier
- Noted that you will be looking for a new icon


Once these changes are in place, we will be good to move this form to prod.

**Comment by RishiRajSahu** [comment]
@acshields 
1. Done
2.Done
3.If didn't found any svg applicable to our usecase so have created two and applied one to the dev env for you to check. the other one is - 
<img width="461" alt="Screenshot 2020-08-17 at 12 07 48 PM" src="https://user-images.githubusercontent.com/11248478/90364785-52ef3480-e082-11ea-88d6-3ce06fe043ed.png">

**Comment by lakshmiathreya** [comment]
1. Screen 2 has Ref Code - is this expected?

**Comment by lakshmiathreya** [comment]
2. What is the expected result for a Copilot who is an employee?

**Comment by lakshmiathreya** [comment]
3. How is the validation done to verify TC employee - is it just the email domain name ?

**Comment by lakshmiathreya** [comment]
Copilot who has a topcoder domain in the email:
<img width="1473" alt="Screenshot 2020-08-31 at 7 20 03 PM" src="https://user-images.githubusercontent.com/51733042/91727260-0d248700-ebbf-11ea-9d17-f0e2b13d088e.png">
<img width="1443" alt="Screenshot 2020-08-31 at 7 20 15 PM" src="https://user-images.githubusercontent.com/51733042/91727271-1150a480-ebbf-11ea-832b-57e455c92444.png">

**Comment by acshields** [comment]
@lakshmiathreya and @RishiRajSahu -

1. REF code is not needed for the internal Topcoder project.
2. Not sure if I completely understand question 2, but here is my thoughts on questions 2/3: If a user has a Topcoder user role in the platform (Manager, Account Executive, Admin, etc.), then they should see this intake form. I suppose this would apply to someone who has the Topcoder user role + Copilot role, but I don't know if there are users with this combination.

**Comment by lakshmiathreya** [comment]
> @lakshmiathreya and @RishiRajSahu -
> 
> 1. REF code is not needed for the internal Topcoder project.
> 2. Not sure if I completely understand question 2, but here is my thoughts on questions 2/3: If a user has a Topcoder user role in the platform (Manager, Account Executive, Admin, etc.), then they should see this intake form. I suppose this would apply to someone who has the Topcoder user role + Copilot role, but I don't know if there are users with this combination.

@acshields the requirement stated above says "Topcoder Employee User" ... could you clarify pls ... Currently all Roles above are being shown/not shown the Internal Topcoder Projects Form as expected - regardless of whether the User is a Topcoder employee or not.
<img width="1297" alt="Screenshot 2020-08-31 at 11 08 34 PM" src="https://user-images.githubusercontent.com/51733042/91749707-89c75d80-ebdf-11ea-97df-29df6f4f483a.png">
![Uploading Screenshot 2020-08-31 at 11.09.56 PM.png…]()

**Comment by RishiRajSahu** [comment]
@lakshmiathreya @acshields I have removed the ref-code field.

@lakshmiathreya regarding visibility of this form, we it is limited users based on the roles, so as mentioned in the issue description people with mentioned roles will be able to see the intake option irrespective of their email-id domain. Also, copilot and general users with just `Topcoder user` role aren't able to see this option.

**Comment by lakshmiathreya** [comment]
-- REF code removed for the Internal Projects form
-- All above roles verified as per specification
